plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -49,8 +49,8 @@ import           PlutusPrelude
 import           Language.PlutusCore.Constant
 import           Language.PlutusCore.Core
 import           Language.PlutusCore.Evaluation.Machine.ExBudgeting
-import           Language.PlutusCore.Evaluation.Machine.ExMemory
 import           Language.PlutusCore.Evaluation.Machine.Exception
+import           Language.PlutusCore.Evaluation.Machine.ExMemory
```

Comment:
I think this change is because `stylish-haskell` changed its opinion on how imports should be sorted.